ray_julia_jll/gen/README.md

```diff
@@ -0,0 +1,25 @@
+# Building and Publishing JLLs
+
+1. Run the `make.jl` script to build the tarball for the host platform and a given julia-version.
+Builds are required for `linux x86_64` and `macos-arm64` on julia `v1.8.0` and `v1.9.0`.
+It is advised you run this within the python virtual environment associated with the root package.
+Note: rerunning this script _will overwrite_ an existing tarball.
+```
+julia --project -e 'using Pkg; Pkg.develop(path=".."); Pkg.instantiate()'
```

Comment:
I think for the `gen` project it would be okay to check in the `Manifest.toml` which would allow you to skip the `Pkg.develop(path="..")` instruction. Doesn't matter much though.

Reply:
done